onchain/rollups/contracts/dapp/ICartesiDApp.sol

```diff
@@ -17,7 +20,7 @@ struct Proof {
 }
 
 /// @title Cartesi DApp interface
-interface ICartesiDApp {
+interface ICartesiDApp is IERC721Receiver, IERC1155Receiver {
```

Comment:
It occurred to me that we should inherit from `IERC721Receiver` and `IERC1155Receiver` as well. So users can more easily detect that we implemented receivers for ERC721 and ERC1155.
And `IERC1155Receiver` inherits from `IERC165` so we don't need to.